src/layout/layout.rs

```diff
@@ -481,6 +475,67 @@ impl Layout {
                         element.size() | EQ(MEDIUM) | f64::from(m),
                     ])?;
                 }
+                Constraint::Length(l) => {
+                    solver.add_constraint(element.size() | EQ(STRONG) | f64::from(l))?
```

Comment:
Question about the strength of Min/Max being higher because we have two constraints (GE and EQ). Would setting the length etc. to STRONG * 2 be worth an experiment?

Like what if all the Mins had an additional weak constraint that was `EQ(WEAK) | area` ? Would this get us somewhere good by default?

Reply:
Typically you want an order of magnitude difference at the very least to be safe. Otherwise you can end up switching behavior when you scale the number of a certain kind of constraints. This can be hard to predict, hard to test and hard to debug. 

I have some ideas for better weights for all the constraints that we can discuss in the other PR.

Reply:
I meant specifically * 2 because we're looking at 2 constraints for the min/max.
Strictly,  STRONG*2 is always > STRONG+MEDIUM

Reply:
Will be discussed here: https://github.com/ratatui-org/ratatui/pull/788